bridge_adaptivity/module/views.py

```diff
@@ -745,6 +743,23 @@ def callback_sequence_item_grade(request):
     return HttpResponse(xml, content_type="application/xml")
 
 
+def _check_and_build_web_socket_message(sequence, score):
```

Comment:
Usually, when the new function is created it is placed upper the place where it is used.

Reply:
Done